Schutzfile

```diff
@@ -2,7 +2,7 @@
   "fedora-36": {
     "dependencies": {
       "osbuild": {
-        "commit": "5eefdc1e9a732a0315c360a3771f1f3ba68a5294"
+        "commit": "d3229dc929d32e927eae4606182c5b02553181ad"
```

Comment:
Don't forget to update the required osbuild version in the SPEC file, once the needed osbuild version is released.